scripts/build_transit_tiles.sh

```diff
@@ -75,7 +75,8 @@ if  [ -n "$TRANSIT_S3_PATH" ]; then
   catch_exception
 
   #push up to s3 the new file
-  aws --region ${REGION} s3 mv ${DATA_DIR}/transit_${stamp}.tgz s3://${TRANSIT_S3_PATH}/ --acl public-read
+  aws --region ${REGION} s3 cp ${DATA_DIR}/transit_${stamp}.tgz s3://${TRANSIT_S3_PATH}/ --acl public-read
+  rm ${DATA_DIR}/transit_${stamp}.tgz
```

Comment:
now catch exception wont fire if there was a problem with the `cp`, it will only care about the `rm`